cmux-tui/bindings/reconcile_registry_artifact.py

```
@@ -410,20 +412,38 @@ def _npm_status(package: str, version: str, artifact: Path) -> str:
+            if (
+                package == NPM_BOOTSTRAP_PACKAGE
+                and latest == NPM_BOOTSTRAP_VERSION
+            ):
+                bootstrap_release = versions.get(NPM_BOOTSTRAP_VERSION)
+                if (
+                    not isinstance(bootstrap_release, dict)
+                    or dist_tags.get("bootstrap") != NPM_BOOTSTRAP_VERSION
+                ):
+                    raise RegistryError(
+                        "npm bootstrap metadata is incomplete for cmux-sdk"
+                    )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)